test/topology_custom/test_tablets.py

```diff
@@ -144,6 +146,28 @@ async def check_allocated_replica(expected: int):
     await check_allocated_replica(rf_to)
 
 
+@pytest.mark.asyncio
+@pytest.mark.xfail(reason="Issue #18786, reading from mutation fragments crashes the node")
```

Comment:
"xfail" means the test will run - and crash - every time. Are we fine with that? I'm worried that we have code which detects cores and announces a failure even despite the xfail marker. I'm also worried about saving a huge core file for every CI run, if we do it.
If any of these is a real problem, you can always use "skip" instead of xfail. This is what I traditionally do in test/cql-pytest for tests that crash the cluster (there it's even more important to skip, because the cluster is shared by all tests so crashing it is a no-no).

By the way, related to the above, it's a shame we don't reuse the same 3-node cluster for multiple tests. As we have more and more tests that just do a trivial request on the cluster and shut it down, it's beginning to look more and more unjustified. I just sent yesterday an example test reusing a 3-node cluster as a fixture.

Reply:
> "xfail" means the test will run - and crash - every time

Correction -- until we fix scylla (#18786). After we do there are two options -- `select from mutation_fragments` would fail or would return something meaningful. In the latter case the xfail can be dropped by the pr that fixes scylla

> Are we fine with that? 

As fine as we are with #18759 . This becomes a "regression" test once the mentioned issue gets fixed.

> If any of these is a real problem, you can always use "skip" instead of xfail. This is what I traditionally do in test/cql-pytest for tests that crash the cluster

Ah, now I see your point. OK, I'll rename it to skip